modules/face/samples/sampleKazemi.cpp

```diff
@@ -0,0 +1,106 @@
+#include<iostream>
```

Comment:
Just to clean code, a space between include and the first `<`. Also on following lines.